src/gui/folderwatcher_mac.cpp

```diff
@@ -105,9 +104,27 @@ void FolderWatcherPrivate::startWatching()
     FSEventStreamStart(_stream);
 }
 
+const QStringList FolderWatcherPrivate::addCoalescedPaths(const QStringList &paths)
```

Comment:
This should return a `QStringList` not `const QStringList` (prevents moves). Also the function itself should be declared const (doesn't modify anything in this AFAICT) or static (doesn't use anything in this AFAICT).

Reply:
Yeah I love const-correctness, but obviously still fail at it regularly 🙄 Thanks.